tests/pages/tests/ga_test.py

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+from selenium.webdriver.common.by import By
+
+from pages.base import BasePage
+
+
+class AnalyticsTestPage(BasePage):
+
+    _URL_TEMPLATE = "/{locale}/external/analytics"
```

Comment:
Issue (blocking): Should this URL be updated too?
```suggestion
    _URL_TEMPLATE = "/{locale}/analytics-tests/"
```